modules/apps/frontend-theme/frontend-theme-unstyled/build.gradle

```diff
@@ -141,28 +140,11 @@ buildClayIcons {
 	into clayIconsDestinationDir
 }
 
-buildLexiconIcons {
```

Comment:
I'm afraid this could be a big breaking change for others outside using the `/lexicon` path.

Let's keep the task in there so we still have the icons at least for this version? We would ideally mark this path as deprecated, but I'm not really sure how to best do that... 

I know it was suggested to place this also in the `BREAKING_CHANGES.md` file, so maybe there? 🤷